### PR TITLE
Add doineedavpn.com to Sources and Further Reading

### PR DESCRIPTION
**Pull request by NylaTheWolf**
## Description

 

I added http://doineedavpn.com/ to the Sources and Further Reading section. It is a quiz by IVPN where the user chooses the issues they want to solve and benefits they look for in their VPN, and informs them whether or not a VPN would actually address any of these issues. 

I think that this website would be a great addition to the Further Reading section because it challenges the misconceptions surrounding VPNs, especially with the aggressive marketing from VPN companies, and helps users decide whether or not they actually need a VPN. It also tells the user if Tor would be a better solution for them. It certainly helped me when I was looking into getting a VPN!

I know that Privacy Guides already addresses these misconceptions, but I still think it's a fantastic source, especially if you wanted to share it with a friend.

### CODE REVIEW

**Comment by ph00lt0** [comment]
Looks good indeed, also nice to see they link to Privacy Guides :)

File: docs/vpn.md [hunk 1]
```diff
@@ -352,6 +352,7 @@ For use cases like these, or if you have another compelling reason, the VPN prov
 3.  [Slicing Onions: Part 1 – Myth-busting Tor](https://medium.com/privacyguides/slicing-onions-part-1-myth-busting-tor-9ec188ae1904) by blacklight447
 4.  [Slicing Onions: Part 2 – Onion recipes; VPN not required](https://web.archive.org/web/20210116140725/https://write.privacytools.io/my-thoughts-on-security/slicing-onions-part-2-onion-recipes-vpn-not-required) by blacklight447
 5.  [IVPN Privacy Guides](https://www.ivpn.net/privacy-guides/)
+6.  ["Do I Need a VPN?"](http://doineedavpn.com/), a tool by IVPN to explain what a VPN does and does not help with and help individual users decide if they need a VPN for their use case.
```
Comment (lx-is) [inline docs/vpn.md:355]:
```suggestion
6.  ["Do I need a VPN?"](https://www.doineedavpn.com/), a tool developed by IVPN to challenge aggressive VPN marketing and help individuals decide if a VPN is right for them.
```

**Review by lx-is — CHANGES_REQUESTED** [review]
It's a good tool, should definitely be added. It might even be better to add it to the "When are VPNs useful?" section instead.

**Comment by dngray** [comment]
I've had a look at this, and am happy to merge.

IVPN produces quality documentation from what I've seen. It's nice for a change, when so many VPN companies just hide behind false marketing.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
IVPN Does link to privacyguides but i get a 404 if i click on it

**Comment by NylaTheWolf** [comment]
Thank you for approving! I'm happy to contribute! 🙂

**Pull request merged**